stores/sql/mysql/main.go

```diff
@@ -877,12 +873,28 @@ func (tx *MainDatabaseTx) UpdateSlab(ctx context.Context, s object.Slab, contrac
 
 func (tx *MainDatabaseTx) UpdateSlabHealth(ctx context.Context, limit int64, minDuration, maxDuration time.Duration) (int64, error) {
 	now := time.Now()
-	healthQuery, args := ssql.HealthQuery(limit, now)
-	durationArgs := []any{now.Add(minDuration).Unix(), maxDuration.Seconds(), minDuration.Seconds()}
-	args = append(args, durationArgs...)
-	res, err := tx.Exec(ctx, fmt.Sprintf("UPDATE slabs sla INNER JOIN (%s) h ON sla.id = h.id SET sla.health = h.health, health_valid_until = (FLOOR(? + RAND() * (? - ?)))", healthQuery), args...)
+	if err := ssql.PrepareSlabHealth(ctx, tx, limit, now); err != nil {
+		return 0, fmt.Errorf("failed to compute slab health: %w", err)
+	}
+
+	args := []any{now.Add(minDuration).Unix(), maxDuration.Seconds(), minDuration.Seconds()}
```

Comment:
we dropped the `append` so we can get rid of the ugly `[]any` here and just inline the args